Added search feature, rounded corners, fixed small details, updated add card look a little

markmiw opened the pull request.

markmiw committed:
Added Search Button features

markmiw committed:
udpate

markmiw committed:
Merge branch 'main' into related_items_start

markmiw committed:
Cleaned up a lot of code and added text compression gzip with additional performance

JeromeMTR commented on client/src/Ratings/NewReviewPages/ReviewPhotos.jsx:
```diff
@@ -1,42 +1,42 @@
+    event.preventDefault();
     props.setPhotos(()=>photoStorage);
-    console.log('submitted photos')
+    console.log('submitted photos');
```
remove console.log

JeromeMTR commented on client/src/Ratings/NewReviewForm.jsx:
```diff
@@ -29,135 +28,137 @@ function NewReviewForm(props) {
-    }
+      data: data,
+    };
     console.log(options);
```
remove console.log

JeromeMTR commented on server/routes/reviews.js:
```diff
@@ -1,35 +1,34 @@
+// set up router middleware
 router.use(function(req, res, next) {
-  console.log(req.url, "@", Date.now());
+  console.log(req.url, '@', Date.now());
```
remove console.log

JeromeMTR reviewed (approved):
Very cool search feature, would love to test it out!

Everything looks great!

Please resolve conflicts and  remove console.log

markmiw committed:
B

markmiw committed:
updated css
Merge branch 'main' of https://github.com/Team-Pacman/Rhydon into related_items_start
